src/main/java/TaskManager.java

```diff
@@ -0,0 +1,251 @@
+                }
+            }
+            else {
+                this.addToList(new Event(input.substring(6, input.length())));
+            }
+            System.out.println("--------------------------------------");
+        }
+
+        else {
+            /*System.out.println("--------------------------------------");
+            this.addToList(new Task(input, TaskType.U));
+            this.taskPrint(input);
+            System.out.println("--------------------------------------");*/
+            throw new DukeException("Oops! There is no such keyword!");
+
+        }
+        int t = toDoList.size();
+        System.out.println("You have a total of " + t + " tasks");
+        System.out.println("--------------------------------------");
+    }
+    //checks whether string is empty after keyword
+    public Boolean checkEmpty(String input, String keyWord) {
+        int keywordLength = keyWord.length();
+        String remainingDescription = input.substring(keywordLength, input.length());
+        if (remainingDescription.length() == 0 ) {
+            return true;
+        }
+        else if (remainingDescription.length() > 1 && remainingDescription.charAt(1) == 32) {
+            return true;
+        }
+        else if (remainingDescription.length() <= 1) {
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
+
+    //returns true if there is a description
+    public Boolean checkPlan(String input) {
+        if(input.contains("/")) {
+            String remainingDescription = input.substring(input.indexOf("/") + 1, input.length());
+            if(remainingDescription.length() != 0) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public void addToList(Task task) {
+        this.toDoList.add(task);
+        save();
+    }
+
+    public void removeFromList(int taskId) {
+        Task tr = this.toDoList.get(taskId - 1);
+        System.out.println("Task successfully removed!");
+        System.out.println("-> " + tr.toString());
+        this.toDoList.remove(taskId - 1);
+        save();
+    }
+
+    public void save() {
+        String path = System.getProperty("user.dir") + "\\" + "save.txt";
+        if(!Files.exists(Paths.get(path))) {
+            try {
+                Files.createFile(Paths.get(path));
+            }
+            catch (IOException e) {
+                System.out.println("error creating file");
+            }
+        }
+        try {
+            new FileWriter(path, false).close();
+        }
+        catch (IOException e) {
+                System.out.println("error deleting file");
+        }
+
+        int i = 1;
+        try {
+            FileWriter myWriter = new FileWriter("save.txt");
+            BufferedWriter out = new BufferedWriter(myWriter);
+            for (Task s : this.toDoList) {
+                    String write = i + ". " + " [" + s.getType() + "] "
```

Comment:
I think there is a wrong indentation on line 208? I believe there should only be 4 spaces indentation.